README.rst

```diff
@@ -0,0 +1,25 @@
+============================================
+buildapi_client
+============================================
+
+Installation
+============
+
+::
+
+    pip install buildapi_client
+
+Documentation
+=============
+
+Usage is described in the documentation:
+http://moz-releng-buildapi.readthedocs.org
```

Comment:
These docs are for BuildAPI itself, so this might be a little confusion. Maybe it would be better to say "This is a Python client for Mozilla's BuildAPI" and link to the docs.